conda/base/context.py

```diff
@@ -120,7 +120,9 @@ class Context(Configuration):
     non_admin_enabled = PrimitiveParameter(True)
 
     # Safety & Security
-    _aggressive_update_packages = SequenceParameter(string_types)
+    _aggressive_update_packages = SequenceParameter(string_types,
+                                                    ('ca-certificates', 'certifi', 'openssl'),
```

Comment:
We should add these as constants in `conda/base/constants.py`, and then use that value as the default here.  Probably just call it `DEFAULT_AGGRESSIVE_UPDATE_PACKAGES`.

Reply:
definitely! +1, done